### PR TITLE
call to get guests of a groop + tests

### PR DESCRIPTION
**Pull request by wlany**
GET groups/{groupId}/guests call implemented + tested

works exactly the same as GET groups/{groupId}/members, but without the host

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GroupController.java [hunk 1]
```diff
@@ -224,6 +224,39 @@ public List<UserGetDTO> getGroupMembersById(@PathVariable Long groupId, HttpServ
...
+        // 401 - not authorized
+        Long tokenId = userService.getUseridByToken(request.getHeader("X-Token"));
+        if(tokenId.equals(0L)) {
+            throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, String.format("You are not authorized."));
+        }
+
+        // get the guests of the group
+        List<Long> guestIds = groupService.getAllGuestIdsOfGroup(group);
+
+        // 204 - no open invitations
```
Comment (akkuebler) [inline src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GroupController.java:243]: is this really checking the inventations? I guess its checking only whether people are in the group?

**Review by akkuebler — APPROVED** [review]
nice and easy - I think this will help Kallia a lot :)

File: src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GroupController.java (hunk 1 above)
Comment (wlany) [reply src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GroupController.java:243]: do you mean invitations? no it doesn't check those at all it just returns the people actually in the group, from our talk I think that is what is needed

Comment (akkuebler) [reply src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GroupController.java:243]: Yeah, spelling mistake. I was confused by the comment "no open INVItations"

**Pull request merged**